### PR TITLE
Updated Roles scripts.

### PR DESCRIPTION
**Pull request by vinayklp**

### CODE REVIEW

**Review by pemmaapps — CHANGES_REQUESTED** [review]
Please make all Modified Date and Modified By columns can be nullables. As first time insertion no record will have these two values.
And make sure all comments and unit test cases are properly implemented.

**Comment by vinayklp** [comment]
> Please make all Modified Date and Modified By columns can be nullables. As first time insertion no record will have these two values.
> And make sure all comments and unit test cases are properly implemented.

**Comment by vinayklp** [comment]
> Please make all Modified Date and Modified By columns can be nullables. As first time insertion no record will have these two values.
> And make sure all comments and unit test cases are properly implemented.

All ModifiedDate and ModifiedBy columns are NULLable...

**Review by pemmaapps — APPROVED** [review]
Take care of sp comments and unit test cases

File: PIFMSMain/Authorization/Procedures/usp_GetRole.sql [hunk 1]
```diff
@@ -1,11 +1,23 @@
 ﻿CREATE PROCEDURE [Authorization].[usp_GetRole]
-	@Name NVARCHAR(50)
+	@Name NVARCHAR(50) = NULL
 AS
-	SELECT 
+	IF(@Name is NULL)
```
Comment (ravindra-pc) [inline PIFMSMain/Authorization/Procedures/usp_GetRole.sql:4]: Check is required ISNULL function. Ex. IF(ISNULL(@name,'') = '')

**Review by ravindra-pc — APPROVED** [review]

**Pull request merged**